.gitignore

```diff
@@ -56,4 +56,5 @@ build-iPhoneSimulator/
 # .rubocop-https?--*
 
 .env
-.rspec_status
\ No newline at end of file
+.rspec_status
+/.idea
```

Comment:
Did you mean to include this?

Reply:
yeah, it's included in stern and this mac file keeps getting added for me